contracts/Uniswap/UniswapConfig.sol

```diff
@@ -25,12 +25,13 @@ contract UniswapConfig {
         PriceSource priceSource;
         uint256 fixedPrice;
         address uniswapMarket;
+        address reporter;
         bool isUniswapReversed;
     }
 
     /// @notice The max number of tokens this contract is hardcoded to support
     /// @dev Do not change this variable without updating all the fields throughout the contract.
-    uint public constant maxTokens = 30;
+    uint public constant maxTokens = 20;
```

Comment:
Reduced the max number of tokens to reduce the size of the resulting contract to fit within the limit. The full complement was not utilized so it was wasted space.